di_website/about/templatetags/count_columns.py

```diff
@@ -0,0 +1,12 @@
+from django import template
+from di_website.vacancies.models import OfficeLocation
+
+
+register = template.Library()
+
+
+@register.simple_tag
+def columns():
+    number_of_offices = OfficeLocation.objects.exclude(
+        latitude__isnull=True).exclude(longitude__isnull=True).all().count()
+    return number_of_offices
```

Comment:
Looks like a good fix @davidebukali but we might want to wait for feedback from @benjamincoleman on what classes are actually meaningful. I'm not sure whether `l-1up` would be valid, and I'm assuming we want to max out at `l-4up`

At which point this line could become: 
```
return max(2, min(number_of_offices, 4))
```
Which would bound it between 2 and 4.

Reply:
Thanks @akmiller01; `return max(2, min(number_of_offices, 4))
` is a neat constraint

Reply:
Just doing a quick search through the pattern library @davidebukali 

Just as I suspected, it starts at 2, but goes to 5. So you could use `max(2, min(number_of_offices, 5))`

Reply:
@akmiller01 this was updated and pushed